File: iFood/FoodCollectionViewCell.m
//
//  FoodCollectionViewCell.m
//  iFood
//
//  Created by Ezequiel Gutierrez on 16/9/16.
//  Copyright © 2016 Acamica. All rights reserved.
//

#import "FoodCollectionViewCell.h"

@interface FoodCollectionViewCell ()

@property (nonatomic, weak) IBOutlet UILabel *titleLabel;
@property (nonatomic, weak) IBOutlet UIImageView *foodImageView;

@end

@implementation FoodCollectionViewCell

- (void) setTile:(NSString*)title andImageName:(NSString*)imageName{
    [self.titleLabel setText:title];
    [self.foodImageView setImage:[UIImage imageNamed:imageName]];
}

@end

File: iFood/Views/FoodCollectionViewCell.swift
//
//  FoodCollectionViewCell.swift
//  iFood
//
//  Created by Matias Fernandez on 10/05/2017.
//  Copyright © 2017 Acamica. All rights reserved.
//

import UIKit

class FoodCollectionViewCell: UICollectionViewCell {
    
    @IBOutlet private weak var titleLabel:UILabel?
    @IBOutlet private weak var foodImageView:UIImageView?

    
    func updateData(_ food: Food) {
        titleLabel?.text = food.name
        foodImageView?.image = UIImage(named: food.name)
    }
    
}
